ds-intermediate-python/who.md

```diff
@@ -0,0 +1,30 @@
+This course is for you if:
+
+* You have been writing software for a while, which may be used by people other than yourself, but it is currently undocumented or unstructured 
+* You want to learn:
+    * more intermediate software engineering techniques and tools
+    * how to collaborate with others to develop software
+    * how to prepare software for others to use
+* You are currently comfortable with:
+    * basic Python programming (though this may not be the main language you use) and applying it to your work on a regular basis
+    * basic version control using Git
+    * command line interface (shell)
+* This course is not for you if:
+    * You have not yet started writing software (in which case have a look at the Software Carpentry course or some other Python course for novices first)
+    * You have learned the basics of writing software but have not applied that knowledge yet (or are unsure how to apply it) to your work. In this case, we suggest you revisit the course after you have been programming for at least 6 months
+    * You are well familiar with the learning objectives of the course and those of individual episodes
+    * The software you write is fully documented and well architected
+
+To attend this course you should meet the following criteria. You can also test your prerequisite knowledge by taking this short quiz.
+
+Git
+* You are familiar with the concept of version control
+* You have experience configuring Git for the first time and creating a local repository
+* You have experience using Git to create and clone a repository and add/commit changes to it and to push to/pull from a remote repository
+* Optionally, you have experience comparing various versions of tracked files or ignoring specific files
```

Comment:
Optional in the sense that this can be acceptable experience instead of some earlier bullet points? Or is it meant more as "Ideally" or "Bonus points if"? :)

Reply:
More as 'ideally'